### DISCUSSION TITLE
How to set max/min width/height of textblock by predefined text

### DISCUSSION THREAD

**Discussion by Intelli-Soft** [unoplatform/uno#8218] [General]
Hey
I am new to UNO.
I want to shift my Winforms project. 
My LOB project will have a lot of textboxes (numeric ones)
So, I want to create a textbox, which has a predefined size, based on the used font size
A very simple example:
Let's say, I have a textbox, which allows to enter the degree of an circle. We know, the maximum is 360°.
So, now the size of the used textbox should have the maximum and minimum width and height set, sothat the value of "360°" is still visible. No matter on which platform I use it.
Could someone please give me advise, how I can create such a control?
Please, I am not talking about setting the minimum/maximum value, which is allowed within the control!!!
I am talking about the size of the control

THX a lot in advance

**Comment by jeromelaban** [comment]
Thanks for the question. How do you do this using Winforms?

**Reply by Intelli-Soft** [reply]
Hey
In WinForms, this is a very easy task.
You just need to use the MeasureString Method ([Graphics.MeasureString](https://docs.microsoft.com/en-us/dotnet/api/system.drawing.graphics.measurestring?view=dotnet-plat-ext-6.0)
And Update the width/height of the control depending on it's result + some predefined margin.

THX a lot
